resources/db/migration/V12__Add Support field.sql

```diff
@@ -0,0 +1,8 @@
+-- adds a new column for storing contact to support for the application
+
+ALTER TABLE zk_data.application
+ ADD  COLUMN a_support_url TEXT;
+
+ALTER TABLE zk_data.application
+ ALTER COLUMN a_support_url SET DEFAULT '';
```

Comment:
please don't set the empty string as default, all other URL fields also have NULL as default